web/src/beta/components/DragAndDropList/Item.tsx

```diff
@@ -15,22 +15,30 @@ type Props = {
   itemGroupKey: string;
   id: string;
   index: number;
+  shouldUseCustomHandler?: boolean;
   onItemMove: (dragIndex: number, hoverIndex: number) => void;
   onItemDropOnItem: (dropIndex: number) => void;
   onItemDropOutside: () => void;
   children: ReactNode;
 };
 
+const ItemContext = createContext<React.RefObject<HTMLDivElement> | null>(null);
+
+export const useItemContext = () => useContext(ItemContext);
```

Comment:
Sorry, similar to the other context, if there isn't a big reason (other than limit prop drilling) for this context can you just pass down as a prop. Later we might want to strategically use context in a refactor to cleanup/simplify the code.

Reply:
@KaWaite the reason am using context here is different from prev one I can explain to you tomorrow on call